docker/docker-start.sh

```diff
@@ -5,6 +5,7 @@ set -e
 # ensure we create the locales first so that Django doesn't complain
 LANGUAGE_CODE=en ./manage.py compilemessages
 
+./manage.py migrate core 0031 --fake
```

Comment:
This is not going to work - for old sites, yes, we need to do this, but for new sites, we can't.
Please remove this and only add it to the `sideloader/postinstall`